testing/test_config.py

```diff
@@ -181,61 +181,66 @@ def test_confcutdir(self, testdir):
     @pytest.mark.parametrize(
         "ini_file_text, invalid_keys, warning_output, exception_text",
         [
-            (
+            pytest.param(
                 """
-          [pytest]
-          unknown_ini = value1
-          another_unknown_ini = value2
-          """,
+                [pytest]
+                unknown_ini = value1
+                another_unknown_ini = value2
+                """,
                 ["unknown_ini", "another_unknown_ini"],
                 [
                     "=*= warnings summary =*=",
                     "*PytestConfigWarning:*Unknown config option: another_unknown_ini",
                     "*PytestConfigWarning:*Unknown config option: unknown_ini",
                 ],
                 "Unknown config option: another_unknown_ini",
+                id="2-unknowns",
```

Comment:
Also made use of `pytest.param(..., id=)` here so the have a recognizable node id.